components/bsx/Offer/OfferTable.vue

```diff
@@ -54,8 +54,9 @@
       v-slot="props"
       sortable>
       <nuxt-link :to="`gallery/${props.row.nft.id}`">
-        <p v-if="props.row.nft.name">{{ props.row.nft.name }}</p>
-        <p v-else>{{ props.row.nft.id }}</p>
+        <p class="limit-width-text">
```

Comment:
can we add `title` here as if the character is more than 50, we wont be able to see the full name if we hover over